From 287674180344886542e04e00b937e426d871a8e1 Mon Sep 17 00:00:00 2001
From: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
Date: Wed, 27 Nov 2019 10:33:29 +0800
Subject: [PATCH] phy: mvebu-cp110-comphy: Suppress the too verbose calltrace
 for the firmware version warning

Reporting calltrace below just because of firmware need to be updated, needless.
-----------[ cut here ]-----------
mvebu-comphy d0018300.comphy: firmware updated needed
WARNING: CPU: 1 PID: 1 at drivers/phy/marvell/phy-mvebu-cp110-comphy.c:472 mvebu_comphy_probe+0x2b0/0x308
Modules linked in:
CPU: 1 PID: 1 Comm: swapper/0 Not tainted 4.18.45-yocto-standard #1
Hardware name: Marvell Armada 3720 Development Board DB-88F3720 (DT)
pstate: 40000005 (nZcv daif -PAN -UAO)
pc : mvebu_comphy_probe+0x2b0/0x308
lr : mvebu_comphy_probe+0x2b0/0x308
sp : ffffff800802bad0
x29: ffffff800802bad0 x28: ffffffc0e266da80
x27: 00000000006080c0 x26: ffffff80089fccb8
x25: ffffffc0e24e6400 x24: ffffff80089fcfb0
x23: ffffff8008b1ae78 x22: ffffffc0e266d580
x21: ffffffc0e32b2c10 x20: ffffffc0e32b2c00
x19: ffffffc0e7feb9b8 x18: 0000000000000000
x17: 0000000000000000 x16: 0000000000000004
x15: ffffffffffffffff x14: ffffff8008cc8688
x13: 0000000000000000 x12: ffffff8008de0000
x11: ffffff8008ce4000 x10: ffffff8008de0100
x9 : 0000000000000001 x8 : ffffff80085e4c50
x7 : 6966203a7968706d x6 : 000000000000008e
x5 : 0000000000000000 x4 : 0000000000000000
x3 : ffffffffffffffff x2 : ffffff8008ce43e0
x1 : 8985a4d09249aa00 x0 : 0000000000000000
Call trace:
mvebu_comphy_probe+0x2b0/0x308
platform_drv_probe+0x58/0xa8
driver_probe_device+0x210/0x2f8
__driver_attach+0xf4/0xf8
bus_for_each_dev+0x84/0xd8
driver_attach+0x30/0x40
bus_add_driver+0x1e0/0x210
driver_register+0x64/0x110
__platform_driver_register+0x54/0x60
mvebu_comphy_driver_init+0x20/0x28
do_one_initcall+0x68/0x248
kernel_init_freeable+0x2d4/0x3cc
kernel_init+0x18/0x104
ret_from_fork+0x10/0x1c
--[ end trace bf5e27f8bfd1914f ]--

Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/phy/marvell/phy-mvebu-cp110-comphy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
index 5aeef2005297..2b1715e7f596 100644
--- a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
+++ b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
@@ -469,7 +469,7 @@ static int mvebu_comphy_probe(struct platform_device *pdev)
 		ret = mvebu_comphy_power_off(phy);
 		if (ret == COMPHY_FW_NOT_SUPPORTED) {
 			dev_warn(&pdev->dev, "RELYING ON BOTLOADER SETTINGS\n");
-			dev_WARN(&pdev->dev, "firmware updated needed\n");
+			dev_warn(&pdev->dev, "firmware updated needed\n");
 
 			/*
 			 * If comphy power off fails it means that the
-- 
2.17.1

